src/db_queries.js

```diff
@@ -15,6 +15,31 @@ dbQueries.getPostData = (connPool, id, callback) => {
   );
 };
 
+dbQueries.getTags = (connPool, id, callback) => {
+  connPool.query(
+    'SELECT tag_id FROM posts_tags WHERE post_id = $1',
+    [id],
+    callback
+  );
+};
+
+dbQueries.getTagNames = (connPool, tagIds, callback) => {
+  let queryString = 'SELECT description FROM tags WHERE ';
+  const dataArray = [];
```

Comment:
You don't need to declare the empty dataArray first. You could just assign it the value of 
``` 
const dataArray = tagIds.map((tag, index, array) => { etc...
)
```